pkg/acsengine/pki.go

```diff
@@ -68,6 +72,18 @@ func CreatePki(extraFQDNs []string, extraIPs []net.IP, clusterDomain string, caP
 
 	go func() {
 		var err error
```

Comment:
The only thing that I'm not sure looks right is if the extra fqdn and extra ios have the right values.